src/main/java/ch/njol/skript/bukkitutil/NamespacedUtils.java

```diff
@@ -0,0 +1,78 @@
+package ch.njol.skript.bukkitutil;
+
+import ch.njol.skript.localization.ArgsMessage;
+import ch.njol.skript.util.ValidationResult;
+import org.bukkit.NamespacedKey;
+
+/**
+ * Utility class for {@link NamespacedKey}
+ */
+public class NamespacedUtils {
+
+	public static final String NAMEDSPACED_FORMAT_MESSAGE =
+		new ArgsMessage("misc.namespacedutils.format").getValue();
```

Comment:
This should just be a Message? Also, I'd store it as Message and have the caller handle `toString()` if necessary